Base/home/anon/.config/Tests.ini

```diff
@@ -1,7 +1,8 @@
 [Global]
-SkipDirectories=Kernel/Legacy UserEmulator
+SkipDirectories=Kernel/Legacy
+SkipRegex=ue-.*
 SkipTests=test-web
-NotTestsPattern=.txt|.frm|.inc
+NotTestsPattern=($.*(txt|frm|inc)^)
```

Comment:
This pattern...won't match anything...
$ is the end-of-line assertion, nothing can match after that

Reply:
Seems like `^` and `$` are flipped. :^)

Reply:
flipped those now